src/dbnode/persist/fs/persist_manager.go

```diff
@@ -654,3 +657,70 @@ func (pm *persistManager) SetRuntimeOptions(value runtime.Options) {
 	pm.currRateLimitOpts = value.PersistRateLimitOptions()
 	pm.Unlock()
 }
+
+var (
+	indexVolumeIndexClaimsLock sync.Mutex
```

Comment:
If we decide on going down the path of a global persist manager instance vs a global shared lock btwn persist managers, currently both flush and cold flush each have their own persist manager instance.

We also create one for the fs/peers bootstrappers but they run serially.

Reply:
Spoke offline, decided on going w/ a global index volume index claims manager approach instead.